web/app/services/auth.service.spec.ts

```diff
@@ -0,0 +1,84 @@
+import {HttpClientTestingModule, HttpTestingController} from '@angular/common/http/testing';
+import {inject, TestBed} from '@angular/core/testing';
+import {mockLoginResponse} from '../common/test_helpers/mock_login_data';
+import {AuthService, LoginRequest, LoginResponse} from './auth.service';
+
+const FAKE_LOGIN_REQUEST: LoginRequest = {
+  username: 'tacoRocat',
```

Comment:
i think the parameter is `email`, but I can change it in the backend. Also, how are you sending this parameters? (`json/application`, `form-data`?)

Reply:
this would be a part of the HTTP request's body.